widgetssdk/src/test/java/com/glia/widgets/messagecenter/MessageCenterControllerTest.kt

```diff
@@ -1,55 +1,107 @@
 
 import com.glia.androidsdk.GliaException
-import com.glia.widgets.core.secureconversations.domain.IsMessageCenterAvailableUseCase
-import com.glia.widgets.core.secureconversations.domain.SendSecureMessageUseCase
+import com.glia.widgets.core.fileupload.model.FileAttachment
+import com.glia.widgets.core.secureconversations.domain.*
 import org.junit.Before
 import org.junit.Test
-import org.mockito.Mockito
-
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.times
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
 
 internal class MessageCenterControllerTest {
     private lateinit var messageCenterController: MessageCenterController
     private lateinit var sendSecureMessageUseCase: SendSecureMessageUseCase
     private lateinit var isMessageCenterAvailableUseCase: IsMessageCenterAvailableUseCase
+    private lateinit var addFileAttachmentsObserverUseCase: AddSecureFileAttachmentsObserverUseCase
+    private lateinit var addFileToAttachmentAndUploadUseCase: AddSecureFileToAttachmentAndUploadUseCase
+    private lateinit var getFileAttachmentsUseCase: GetSecureFileAttachmentsUseCase
+    private lateinit var removeFileAttachmentObserverUseCase: RemoveSecureFileAttachmentObserverUseCase
+    private lateinit var removeFileAttachmentUseCase: RemoveSecureFileAttachmentUseCase
     private lateinit var viewContract: MessageCenterContract.View
 
     @Before
     fun setUp() {
-        sendSecureMessageUseCase = Mockito.mock(SendSecureMessageUseCase::class.java)
-        isMessageCenterAvailableUseCase = Mockito.mock(
-            IsMessageCenterAvailableUseCase::class.java
-        )
+        sendSecureMessageUseCase = mock()
+        isMessageCenterAvailableUseCase = mock()
+        addFileAttachmentsObserverUseCase = mock()
+        addFileToAttachmentAndUploadUseCase = mock()
+        getFileAttachmentsUseCase = mock()
+        removeFileAttachmentObserverUseCase = mock()
+        removeFileAttachmentUseCase = mock()
+        viewContract = mock()
         messageCenterController =
-            MessageCenterController(sendSecureMessageUseCase, isMessageCenterAvailableUseCase)
-        viewContract = Mockito.mock(MessageCenterContract.View::class.java)
+            MessageCenterController(sendSecureMessageUseCase, isMessageCenterAvailableUseCase,
+                addFileAttachmentsObserverUseCase, addFileToAttachmentAndUploadUseCase,
+                getFileAttachmentsUseCase, removeFileAttachmentObserverUseCase,
+                removeFileAttachmentUseCase)
+    }
+
+    @Test
+    fun setView_ExecutesAddSecureFileAttachmentsObserverUseCase_onTrigger() {
+        messageCenterController.setView(viewContract)
+
+        verify(addFileAttachmentsObserverUseCase, times(1)).execute(any())
+    }
+
+    @Test
+    fun setView_ExecutesGetFileAttachmentsUseCase_onTrigger() {
         messageCenterController.setView(viewContract)
+
+        verify(getFileAttachmentsUseCase, times(1)).execute()
```

Comment:
it is possible to remove `times(1)`, 

```
public static <T> T verify(T mock) {
        return MOCKITO_CORE.verify(mock, times(1));
    }
```

Reply:
Yes. But previously we decided it is a more clear way to write the verification rules.